be/src/exec/aggregator.cpp

```diff
@@ -136,7 +136,7 @@ Status Aggregator::open(RuntimeState* state) {
         RETURN_IF_UNLIKELY_NULL(_single_agg_state, Status::MemoryAllocFailed("alloc single agg state failed"));
         auto call_agg_create = [this]() {
             for (int i = 0; i < _agg_functions.size(); i++) {
-                _agg_functions[i]->create(_agg_fn_ctxs[0], _single_agg_state + _agg_states_offsets[i]);
+                _agg_functions[i]->create(_agg_fn_ctxs[i], _single_agg_state + _agg_states_offsets[i]);
```

Comment:
it seems a bug? need to cp to other branch ?

Reply:
maybe previous agg functions's create() does need some info from the ctx.